apps/polyflix/src/modules/core/components/Footer/footer.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { Paper, Typography } from '@mui/material'
+import { PageSection } from '../PageSection/page-section.component'
+
+export const Footer = () => {
+  return (
+    <Paper
+      variant="outlined"
+      component={'footer'}
+      sx={{
+        minHeight: '200px',
+        p: 2,
+        mt: 4,
+      }}
+    >
+      {/* TODO */}
```

Comment:
No footer for the moment ?

Reply:
It's juste a placeholder for a second PR